.github/workflows/old-release.yml

```diff
@@ -0,0 +1,206 @@
+env:
```

Comment:
I assume this is mostly from the state of the repo as of 0.40 release and the makefile.old is verbatim are mostly copied from the 0.40.x release/branch?